packages/core/src/App/Containers/Layout/header/__tests__/traders-hub-home-button.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import TradersHubHomeButton from '../traders-hub-home-button';
+
+jest.mock('react-router', () => ({
+    ...jest.requireActual('react-router'),
+    useHistory: () => ({ location: { pathname: '/appstore/traders-hub' } }),
+}));
+
+describe('TradersHubHomeButton', () => {
+    it("should display the text Trader's Hub in the header", () => {
+        render(
+            <StoreProvider store={mockStore({})}>
+                <TradersHubHomeButton />
+            </StoreProvider>
+        );
+        expect(screen.getByText("Trader's Hub")).toBeInTheDocument();
+    });
+
+    it('should have the --active class if in traders hub route', () => {
```

Comment:
Do we need this testcase as it is checking CSS class presence

Reply:
yes, this testcase makes sure that the traders hub link is highlighted when user is in taders hub page. this test ensures that an important visual feedback is working.